resources/install/debian/rules.tmpl

```diff
@@ -31,6 +37,16 @@ ifeq ($(DEB_HOST_ARCH),amd64)
 else
 	dh_install -p$(PACKAGE_NAME)-jni lib/native/linux/*.so usr/lib/jni/
 endif
+	test -e $(USR_LIB_FOLDER)/libavformat-ffmpeg.so.56 && dh_link $(USR_LIB_FOLDER)/libavformat-ffmpeg.so.56 usr/lib/jni/libavformat.so || true
```

Comment:
I assume usr/lib/jni is local to Jitsi and not the global /usr/lib? Otherwise this might create a huge mess...

Reply:
We are doing a replace for the actual path:
https://github.com/jitsi/jitsi/blob/07df10c510c1a23b30cf31c2d4f9db1a9cb23b65/resources/install/build.xml#L2461

This is because of the source package we used to build for debian official release. Maybe later we can get rid of all that mess, as nobody will be working on that and anyway this is not the right path for it.

Reply:
Ah, the template 🤦‍♂️